7_deploy_osh/roles/suse-osh-deploy/tasks/main.yml

```diff
@@ -174,6 +184,27 @@
     - keystone
     - run
 
+- name: Use a different horizon script as missing a multinode version without node port requirement
```

Comment:
Aha! Could you please put that in a comment in the script itself, so we know that socok8s has added it and what the origin of the file is in case we need to update it later?

Reply:
Will do that